cli/cmdline/debug/show/debug-show-generate-origins.md

```diff
@@ -0,0 +1,29 @@
+---
+title: terramate debug show generate origins - Command
+description: Debug generated files using the 'terramate debug show generate-origins' command to gain insights into the origin of each file.
+---
+# Generate Origins
+
+The `terramate debug show generate-origins` command prints the origin source for the generated files.
+
+## Usage
+
+`terramate debug show generate-origins [options]`
+
+## Examples
+
+Print the origins of all generated files in the current directory:
+
+```bash
+terramate debug show generate-origins
+```
+Change the working directory before listing origins:
+
+```bash
+terramate debug show generate-origins --chdir path/to/directory
```

Comment:
```suggestion
terramate --chdir path/to/directory debug show generate-origins 
```
reasoning: `--chdir` is a global flag, so better to move it to beginning so it's not confused for being a "generate-origins" special flag.

Additionally, but this is minor, in every other example we use `-C <path>` (the short flag), so maybe makes sense to stick to one style.